### DISCUSSION TITLE
Mentions on Mobile

### DISCUSSION THREAD

**Discussion by davevilela** [sodenn/lexical-beautiful-mentions#516] [Q&A]
Description:

I am currently working on a proof of concept using Lexical and React Native. To render the editor, I am utilizing a WebView and managing the communication between the host and client with the @webview-bridge/react-native library.

Issue:

I am trying to implement mentions/suggestions so that the popover displaying query results appears in a native view above the input. I need to find a way to use this plugin to send the state of the query back to React Native after the trigger.

Question:

Is there a way to leverage lexical-beautiful-mentions to report the state of the query to React Native after the trigger is activated?

**Comment by sodenn** [comment]
Hi @davevilela. I am not a React Native expert, but would it be possible to send the query data to React Native in the `onSearch` method? `onSearch` is executed immediately after entering a trigger character. Note: If you use the `items` prop, you must switch to `onSearch` (you cannot use both props at the same time).

Important: Android's keyboard does not work well with contenteditable. The editor feels buggy with the Gboard keyboard (e.g., https://github.com/facebook/lexical/issues/4636). Unfortunately, these bugs are not easy to fix. So, consider carefully whether an editor based on contenteditable is really suitable for you if you know that many Android users will use the app.
